### ISSUE TITLE
Can't boot into `bluefin-dx-surface`

### ISSUE THREAD

**Issue by darribas** [ublue-os/surface#35]
Hello,

I'm on Surface Book 2 where I flashed the universal blue iso and installed vanilla Silverblue following:

https://universal-blue.org/installation/

So far so good, I can encrypt the disk and boot into the desktop, looks great (though no touch enabled). I now want to switch to the `bluefin-dx-surface`, so I follow instructions on:

https://universal-blue.org/images/

I first rebase to an unsigned image, then reboot, rebase to the signed one and reboot. I can now see bluefin on the GRUB but, when I try to boot into it, I get a black screen with only two messages:

```
error: ../../grub-core/kern/efi/sb.c:182:bad shim signature
error: ../../grub-core/loader/i386/efi/linux.c:258:you need to load the kernel first
```

Any idea what's going on and how I could work around it to boot it? So far, the Silverblue experience has been so good, if I could get the surface flavour to get all the bits (e.g., touch screen) working, it'd be incredible!

**Comment by castrojo** [comment]
I think an error like this is usually secure boot related?

**Comment by darribas** [comment]
Yes, that's my sense. From reading a bit more on the `linux-surface` project, it appears it needs to sign a key in a manual process that I guess I must have missed?

https://github.com/linux-surface/linux-surface/issues/821

Any suggestions on how to re-run the install or make it work?

**Comment by pesader** [comment]
Hi, @darribas!

I had the same issue and got some help at the Discord server, which solved it for me. What you to do is enroll the MOK key that is used by the UBlue project in your firmware.

To do that, run:

```bash
mokutil --import /etc/pki/kernel/private/private_key.priv
```

If asked for a password, simply type "ublue-os". Then reboot your computer and choose to "Enroll MOK key". Eventually you will be asked for the password that you set up in your terminal ("ublue-os") so type that out and you're good to go :)
